scripts/genetic_backtester/darwin.js

```diff
@@ -1045,18 +1045,18 @@ function saveLaunchFiles(saveLauchFile, configuration ){
   //write Nix Version
   let lNixContents = '#!/bin/bash\n'.concat('#fitness=',configuration.fitness,'\n',
     'env node zenbot.js trade ', 
-    bestOverallCommand,'\n')
+    bestOverallCommand,' $@\n')
   let lWin32Contents = '@echo off\n'.concat('rem fitness=',configuration.fitness,'\n',
     'node zenbot.js trade ', 
-    bestOverallCommand,'\n')
+    bestOverallCommand,' %*\n')
   
   if (Number(configuration.fitness) > Number(lastFitnessLevel))
   {
     fs.writeFileSync(lFilenameNix, lNixContents)
     fs.writeFileSync(lFinenamewin32, lWin32Contents)
     // using the string instead of octet as eslint compaines about an invalid number if the number starts with 0
-    fs.chmodSync(lFilenameNix, 0777)
-    fs.chmodSync(lFinenamewin32, 0777)
+    fs.chmodSync(lFilenameNix, '777')
+    fs.chmodSync(lFinenamewin32, '777')
```

Comment:
not `'0777'`?

Reply:
According to all the docs I’ve read and checking the FS source, it takes 2 types of input. a number represented as octets (0777 the zero in front designating it as an octet number) or a string. 
the string being a literal representation of the octet value of 777.  it sends the string thru a conversion, which results in 0777 number or 511 decimal or 1FF hex, depending on how you view it. 

I would have left it as 0777 since it eliminates the internal conversion but .eslint complains that 0777 is an “invalid number” and my post checks eslint errors and prevents posts if there are any, and since this isn’t a tight loop area, the extra conversion doesn’t hurt, so decided on the string route. 

This link started my research on it once I found out I made the mistake, and why it is the way it is. https://stackoverflow.com/questions/8756639/how-do-i-use-chmod-with-node-js  from there it went to the FS source and tracking it on thru.

Reply:
Yeah, that's a deprecated octal format. It seems like there's a couple solutions, but the current changes likely work. ref: https://github.com/babel/babel/issues/2569

Reply:
Ty.  I didn't realize about the 0o designation.